app/scripts/metamask-controller.js

```diff
@@ -2359,6 +2359,9 @@ export default class MetamaskController extends EventEmitter {
         [MetaMetricsUserTrait.NftAutodetectionEnabled]: useNftDetection,
       },
     });
+
+    // set up sentry after onboarding if opt in metrics(this preference will be validated in `startSession`
+    window.sentry?.startSession();
```

Comment:
Since we're in the background, should this be `globalThis.sentry` for consistency?

Reply:
makes sense. Both methods works but will update to `globalThis.sentry` [1b9a62](https://github.com/MetaMask/metamask-extension/pull/25990/commits/1b9a62b2bc3cee88dbe709a49739805082a11fc8)